doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3134,12 +3134,12 @@ Build system
 
 The issues in this section are related to :ref:`app_build_system`.
```

Comment:
Please add a new issue per https://github.com/nrfconnect/sdk-nrf/pull/19292
.. rst-class:: v2-8-0 v2-7-0

NCSDK-30458: The BLE applications build with the `nordic-bt-rpc` snippet don't work on nRF54H20 devices.
  When using the ``nordic-bt-rpc`` sysbuild snippet for building BLE applications for nRF54H20 in the Bluetooth RPC configturation, the build system creates an invalid memory map for the application core resulting in a runtime failure.

**Workaround:** Add the `status = "disabled";` line in the `cpurad-rw-partitions` node definition in the :file:`snippets/nordic-bt-rpc/boards/nrf54h20dk_nrf54h20-mem-map-move.dtsi` file.

Reply:
Updated.